lib/parser/onnx/onnx_parser.cc

```diff
@@ -333,10 +333,6 @@ Tensor<T> ONNXParser::ProcessTensor(const onnx::TensorProto& tensor_proto) {
     LOG(ERROR) << "Unsupported external data storage.";
   }
 
-  if (shape.empty() && !v.empty()) {
```

Comment:
If the v.size() > 1, should we set the shape?

Reply:
yes, you are right, we should check shape.empty && v.size() > 1